src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py

```diff
@@ -294,6 +294,7 @@ def convert_to_habitat_obs_space(
             }
         )
 
+    # FIXME: the return values do not match the signature
```

Comment:
@yvsriram @JHurricane96 - does this seem right to you?

Reply:
Yes, this is right. Will fix this one in our integration PR today